app/packs/src/apps/admin/devices/DeviceUserGroupsTab.js

```diff
@@ -36,8 +35,8 @@ const DeviceUserGroupsTab = () => {
 
   return (
     <Form>
-      <FormGroup>
-        <ControlLabel>Add device permission to users</ControlLabel>
+      <Form.Group className="mb-4">
+        <Form.Label className="fw-bold">Add device permission to users</Form.Label>
```

Comment:
Looking at your code has convinced me to overwrite the default font weight of labels. They are now bold by default. Please remove the fw-bold classes from the labels.